### DISCUSSION TITLE
Adding more than 1 owner of one difficulty (GD)

### DISCUSSION THREAD

**Discussion by ghost** [ppy/osu#24096] [Ideas] [answered]
Perhaps it could be a good idea to add the limit for adding GD owner from 1 to 5 GD owners of limit in one difficulty, not sure if someone already asked this but i feel like it would be nice if there was such a thing, since there is some people who make collab GD more than 2 people in one difficulty so i thought it would be relief if this is gonna happen one day.

**Comment by ghost** [comment]
If there is already the similar question about it, i apologize about it.

**Reply by Joehuu** [reply, marked as answer]
See https://github.com/ppy/osu-web/issues/7602.

**Reply by ghost** [reply]
Thank you.
